Core/GDCore/Events/Event.h

```diff
@@ -139,6 +140,21 @@ class GD_CORE_API BaseEvent {
     return noExpr;
   };
 
+  /**
+   * \brief Return a list of all expressions with ParameterMetadata of the event.

```

Comment:
Return a list of all expressions of the event, each with their associated metadata.